samples/data-sealing/common/CMakeLists.txt

```diff
@@ -9,15 +9,27 @@ add_custom_command(OUTPUT datasealing_t.h datasealing_t.c datasealing_args.h
 # Create a library common to each of our three enclaves.
 add_library(common STATIC dispatcher.cpp keys.cpp ${CMAKE_CURRENT_BINARY_DIR}/datasealing_t.c)
 target_compile_definitions(common PUBLIC OE_API_VERSION=2)
-target_link_libraries(common PUBLIC
-  # `liboecore`, a dependency of `liboeenclave`, requires the ecalls
-  # function table. Because the libraries linking `libcommon` do not
-  # directly require this symbol, the linker skips the object in
-  # `libcommon` which defines them. So we manually require it.
-  #
-  # Alternatively we could use a CMake OBJECT library, but that
-  # requires a newish version of CMake.
-  -Wl,--require-defined=__oe_ecalls_table
-  openenclave::oeenclave
-  openenclave::oelibcxx)
+
+if(NOT WIN32)
+  target_link_libraries(common PUBLIC
+    # `liboecore`, a dependency of `liboeenclave`, requires the ecalls
+    # function table. Because the libraries linking `libcommon` do not
+    # directly require this symbol, the linker skips the object in
+    # `libcommon` which defines them. So we manually require it.
+    #
+    # Alternatively we could use a CMake OBJECT library, but that
+    # requires a newish version of CMake.
+    -Wl,--require-defined=__oe_ecalls_table
```

Comment:
Since this is the only difference, can we just use a generator expression to make it conditionally off on Windows? It'd be a lot less code.

Reply:
`$<$<PLATFORM_ID:Linux>:-Wl,--require-defined=__oe_ecalls_table>`

Reply:
Nice simplification, thanks Andy! Added to PR